src/components/MessageComponent.vue

```diff
@@ -32,4 +32,25 @@ defineProps({
     required: true
   },
 });
+
+const processedMessage = computed(() => {
```

Comment:
toto robi co?

Reply:
@tomasdrga zvyrazni ked oznacis niekoho v texte, je to riesene takto, riadok 37 nahradza znaky < > cim sa zabranuje xss cize hocijaky HTML kod sa zobrazi ako text, riadok 39 tam je regex na zaklade ktoreho sa zvyraznia oznacenia "@username" tym ze sa obalia do tagu <mark> ktory ma prideleny style dole

Reply:
jasne rozumiem, good job, zatial to za mna vyzera good